examples/optimize_triplet/input_triplet.in

```diff
@@ -0,0 +1,57 @@
+###############################################################################
+# Particle Beam(s)
+###############################################################################
+beam.npart = 10000
+beam.units = static
+beam.kin_energy = 2.0e3
```

Comment:
2 GeV initialized here - should that read 1 GeV?

Reply:
The discrepancy was in the README.  The correct value is 2 GeV, and the README has been updated.